plugins/backend-trace-forwarder/src/Cardano/BM/Backend/TraceForwarder.lhs

```diff
@@ -295,15 +306,13 @@ sendItems config tfTVar h items@(lo:_) =
           return () -- Everything is ok, LogObjects were written to the handler.
         Left (_e :: IOException) -> do
           -- Handle is bad, it looks like the connection is already broken.
-          -- Remove bad handle to initiate reconnection.
-          -- Old handle will be removed later by GC.
+          -- Remove bad handle, please see Note [Handle and GC].
```

Comment:
Very small nit (here and elsewhere):

`Remove bad handle,` -> `Remove bad handle to initiate reconnection.` -- the Note is mainly about GC safety.

Reply:
Fixed.